lib/matplotlib/sphinxext/plot_directive.py

```diff
@@ -600,6 +635,27 @@ def _parse_srcset(entries):
     return srcset
 
 
+def check_output_base_name(env, output_base):
+    docname = env.docname
+
+    if '.' in output_base or '/' in output_base:
+        raise PlotError(
+            f"The image-basename '{output_base}' is invalid. "
+            f"It must not contain dots or slashes.")
+
+    for d in env.mpl_plot_image_basenames:
+        if output_base in env.mpl_plot_image_basenames[d]:
+            if d == docname:
+                raise PlotError(
+                    f"The image-basename "
+                    f"{output_base}' is used multiple times.")
```

Comment:
Fits in a line; also the quotes are unbalanced:
```suggestion
                    f"The image-basename {output_base!r} is used multiple times.")
```